ebuild-writing/file-format/text.xml

```diff
@@ -19,9 +19,30 @@ An ebuild should be named in the form <c>name-version.ebuild</c>.
 </p>
 
 <p>
-The name section should contain only lowercase non-accented letters, the digits
-0-9, hyphens, underscores and plus characters. Uppercase characters are strongly
-discouraged, but technically valid.
+The name section should mimic the identifier most frequently used by the user in a
```

Comment:
Me nitpicking but I think you're putting too much information in a single sentence. One sentence for allowed chars, one sentence about what should it resemble. Also 'non-Gentoo' context sounds weird… maybe just 'in context to the package' or something like that. It's obvious that the name isn't supposed to resemble itself ;-P.